mmdet/utils/misc.py

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import glob
+import os.path as osp
+
+
+def find_latest_checkpoint(path, ext='pth'):
+    """
+    References:
+        .. [1] https://github.com/microsoft/SoftTeacher
+                  /blob/main/ssod/utils/patch.py
+    """
+    if not osp.exists(path):
```

Comment:
Can not simply return under abnormal conditions, but output relevant information.